include/openssl/types.h

```diff
@@ -33,6 +33,12 @@ extern "C" {
 # include <openssl/safestack.h>
 # include <openssl/macros.h>
 
+#if OPENSSL_VERSION_MAJOR >= 4
+# define OSSL_CONST_FUTURE const
+#else
+# define OSSL_CONST_FUTURE
+#endif
```

Comment:
This needs more indentation by one space. Sorry for not noticing that earlier.

Reply:
Oh right, fixed as well.